lib/linux_kernel/aarch64_cycle_counter.c

```diff
@@ -0,0 +1,87 @@
+/* -*- C -*- */
+/*
+ * Copyright (c) 2021 Seagate Technology LLC and/or its Affiliates
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * For any questions about this software or licensing,
+ * please email opensource@seagate.com or cortx-questions@seagate.com.
+ *
+ */
+
+
+#define M0_TRACE_SUBSYSTEM M0_TRACE_SUBSYS_LIB
+
+#include "lib/trace.h"
+#include <linux/kernel.h>
+#include <linux/module.h>
+#include <linux/version.h>
+
+/* Module for ARM64 linux platform to provide
+** user space access of perf monitor registers*/
```

Comment:
How does userspace access them? Please elaborate it a bit more.

Reply:
It is part of kernel function . never called in user space

Reply:
The Doxygen comments looks like:
```
/**
 * Some comments for a function.
 */
```

Please follow this style.

Reply:
added the comment accordingly

Reply:
I don't see this is fixed.